frontend/public/components/resource-dropdown.tsx

```diff
@@ -71,10 +71,12 @@ const ResourceListDropdown_: React.SFC<ResourceListDropdownProps> = (props) => {
       }
 
       // Only show preferred version for resources in the same API group.
-      const preferred = (m: K8sKind) =>
-        preferredVersions.some(
-          (v) => v.groupVersion === apiVersionForReference(referenceForModel(m)),
-        );
+      const preferred = (m: K8sKind) => {
+        const grp = groupToVersionMap?.[m.apiGroup];
```

Comment:
Does this work correctly for "core" resources not in an API group?

Reply:
AFAIK core groups do not have a preferred version.